src/lib/EntryPointAccess.ts

```diff
@@ -0,0 +1,43 @@
+import { TypedJSON, jsonArrayMember, jsonObject } from "typedjson";
+
+export abstract class EntryPointAccess {
+    public abstract toJSON(): unknown
+}
+
+export class PublicEntryPointAccess extends EntryPointAccess {
+    public toJSON(): unknown {
```

Comment:
```suggestion
    public toJSON(): string {
```

Reply:
I think its okay `string` for PUblicEntryPointAccess, TemplateEntryPointAccess , in SDK, we should avoid using unkown or any as possible, but we can do this later.